add option babeld_over_librenet6

aparcar opened the pull request:
make babeld run on librenet6 tinc interface

G10h4ck commented:
seems that babel+uci is pretty sensitive to order of config sections, please move this before redistribute rules, moreover i would specify that the interface is tunnel if possible (check if the script which convert from the UCI file to babel config support that option)

aparcar commented:
Moved it up before the redistribute statements. Also added `type tunnel`. The docs mention if not added, it's detected automatically, maybe we shoudl trust on that?

G10h4ck commented:
I have tested the autodetection and it wasn't reliable for vlan interfaces, so better to specify when we know for sure
